policies/api/http/endpoint_test.go

```diff
@@ -30,10 +30,14 @@ import (
 )
 
 const (
-	token       = "token"
-	email       = "user@example.com"
-	format      = "yaml"
-	policy_data = `version: "1.0"
+	token               = "token"
+	validPolicyDataJson = "{ \"name\": \"simple_dns\", \"backend\": \"pktvisor\", \"policy\": { \"version\": \"1.0\", \"visor\": { \"policies\": { \"mydefault\": { \"input\": {\"tap\": \"mydefault\"}, \"handlers\": {\"modules\": {\"default_net\": {\"type\": \"net\"},\"default_dns\": {\"type\": \"dns\"}}}}}}}}"
+	conflictValidJson   = "{ \"name\": \"my-policy-conflict\", \"description\": \"A policy example\", \"tags\": { \"region\": \"eu\", \"node_type\": \"dns\" }, \"backend\": \"pktvisor\", \"policy\": {\n  \"version\": \"1.0\",\n  \"visor\": {\n    \"taps\": {\n      \"anycast\": {\n        \"type\": \"pcap\",\n        \"config\": {\n          \"iface\": \"eth0\"\n        }\n      }\n    }\n  }\n}}"
+	validPolicyDataYaml = `{"name": "mypktvisorpolicyyaml-3", "backend": "pktvisor", "description": "my pktvisor policy yaml", "tags": {"region": "eu", "node_type": "dns"}, "format": "yaml","policy_data": "version: \"1.0\"\nvisor:\n    foo: \"bar\""}`
```

Comment:
let's make "policy_data" in the yaml this: `handlers:\n  modules:\n    default_dns:\n      type: dns\n    default_net:\n      type: net\ninput:\n  input_type: pcap\n  tap: mydefault\nkind: collection`